includes/class-gp-translation-helpers.php

```diff
@@ -349,7 +349,7 @@ public function reject_with_feedback() {
 		$locale_slug       = $helper_discussion->sanitize_comment_locale( sanitize_text_field( $_POST['data']['locale_slug'] ) );
 		$translation_id    = $helper_discussion->sanitize_translation_id( intval( $_POST['data']['translation_id'] ) );
 		$original_id       = $_POST['data']['original_id'];
-		$reject_reason     = ! empty( $_POST['data']['reason'] ) ? $_POST['data']['reason'] : '';
+		$reject_reason     = ! empty( $_POST['data']['reason'] ) ? $_POST['data']['reason'] : array( 'other' );
```

Comment:
I think it would be interesting to sanitize the values of the array:

```suggestion
		$reject_reason     = ! empty( $_POST['data']['reason'] ) ? $_POST['data']['reason'] : array( 'other' );
		$reject_reason     = array_map( 'sanitize_text_field', $reject_reason );
```

Reply:
Yea, updated, thanks!

Reply:
Isn't this actually a defined list of reasons? For some reason I thought we already have a check against that but I mistook `sanitize_comment_topic` for that. Could we add a `sanitize_reject_reason`?